package/yast2-drbd.spec

```diff
@@ -15,34 +15,26 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-%define _fwdefdir %{_libexecdir}/firewalld/services
 
+%define _fwdefdir %{_libexecdir}/firewalld/services
 Name:           yast2-drbd
-Version:        4.3.1
+Version:        4.3.2
 Release:        0
 Summary:        YaST2 - DRBD Configuration
 License:        GPL-2.0-or-later
 Group:          System/YaST
-Url:            https://github.com/yast/yast-drbd
-
+URL:            https://github.com/yast/yast-drbd
 Source0:        %{name}-%{version}.tar.bz2
 Source1:        drbd.firewalld.xml
-
+BuildRequires:  firewall-macros
 BuildRequires:  ruby
 BuildRequires:  update-desktop-files
 # SuSEFirewall2 replaced by Firewalld(fate#323460)
 BuildRequires:  yast2 >= 4.0.39
 BuildRequires:  yast2-devtools >= 4.2.2
-BuildRequires:  firewall-macros
-
-# SuSEFirewall2 replaced by Firewalld(fate#323460)
-Requires:       yast2 >= 4.0.39
```

Comment:
This dependency should not be removed, this package uses many modules (like `Popup`, `Mode`, ...) from the base `yast2` package, it would not work without it.

Reply:
thanks.